src/WCSimPhysicsList.cc

```diff
@@ -392,349 +369,341 @@ void WCSimPhysicsList::ConstructGeneral()
 	  }
 
-	  G4HEProtonInelastic* theHEInelasticModel = new G4HEProtonInelastic;
-	  theInelasticProcess->RegisterMe(theHEInelasticModel);
+	  // Use FTFP model - FDL
+	  FTFP_model->SetMinEnergy(4.0*CLHEP::GeV);
+	  theInelasticProcess->RegisterMe(FTFP_model);
 	  pmanager->AddDiscreteProcess(theInelasticProcess);
 	}
 
-      else if (particleName == "anti_proton") 
+      else if (particleName == "anti_proton")
 	{
 	  pmanager->AddDiscreteProcess(theElasticProcess);
-	  G4AntiProtonInelasticProcess* theInelasticProcess = 
+	  G4AntiProtonInelasticProcess* theInelasticProcess =
 	    new G4AntiProtonInelasticProcess();
-	  G4LEAntiProtonInelastic* theLEInelasticModel = 
-	    new G4LEAntiProtonInelastic;
-	  theInelasticProcess->RegisterMe(theLEInelasticModel);
-	  G4HEAntiProtonInelastic* theHEInelasticModel = 
-	    new G4HEAntiProtonInelastic;
-	  theInelasticProcess->RegisterMe(theHEInelasticModel);
+	  theInelasticProcess->RegisterMe(FTFP_model);
 	  pmanager->AddDiscreteProcess(theInelasticProcess);
 	}
 
-      else if (particleName == "neutron") 
+      else if (particleName == "neutron")
 	{
 	  // elastic scattering
-	  G4HadronElasticProcess* theNeutronElasticProcess = 
+	  G4HadronElasticProcess* theNeutronElasticProcess =
 	    new G4HadronElasticProcess;
-	  G4LElastic* theElasticModel1 = new G4LElastic;
+	  G4HadronElastic* theElasticModel1 = new G4HadronElastic;
 	  G4NeutronHPElastic * theElasticNeutron = new G4NeutronHPElastic;
 	  theNeutronElasticProcess->RegisterMe(theElasticModel1);
-	  theElasticModel1->SetMinEnergy(19*MeV);
+	  theElasticModel1->SetMinEnergy(19*CLHEP::MeV);
 	  theNeutronElasticProcess->RegisterMe(theElasticNeutron);
 	  G4NeutronHPElasticData * theNeutronData = new G4NeutronHPElasticData;
 	  theNeutronElasticProcess->AddDataSet(theNeutronData);
 	  pmanager->AddDiscreteProcess(theNeutronElasticProcess);
-	  
+
 	  // inelastic scattering
 	  G4NeutronInelasticProcess* theInelasticProcess =
 	    new G4NeutronInelasticProcess();
-	  
-	  //=================================
-	  // Added by JLR 2005-07-05
-	  //=================================
-	  // Options for secondary interaction models
-	  // Choice defined in jobOptions.mac, which is
-	  // read in before initialization of the run manager.
-	  // In the absence of this file, BINARY will be used.
-	  // GHEISHA = Original Geant4 default model 
-	  // BERTINI = Bertini intra-nuclear cascade model
-	  // BINARY  = Binary intra-nuclear cascade model
-	  if (gheishahad) {
-	    G4LENeutronInelastic* theInelasticModel = new G4LENeutronInelastic;
-	    theInelasticModel->SetMinEnergy(19*MeV);
-	    theInelasticProcess->RegisterMe(theInelasticModel);
-	  }
-	  else if (bertinihad) {
+	  if (bertinihad) {
 	    G4CascadeInterface* theBertiniModel = new G4CascadeInterface;
-	    theBertiniModel->SetMinEnergy(19*MeV);
+	    theBertiniModel->SetMinEnergy(19*CLHEP::MeV);
+	    theBertiniModel->SetMaxEnergy(5.0*CLHEP::GeV);
```

Comment:
No, it is not a requirement at this juncture.  The idea is to have a better practice to understand exactly which model is valid where, especially if things change as GEANT makes new releases.  We're not quite as beholden if they change a threshold in that case (and it may help validations).